### DISCUSSION TITLE
How to write `_document.js` as a functional component?

### DISCUSSION THREAD

**Discussion by wiktoriavh** [vercel/next.js#23139] [Help]
As the title says. I want to integrate my own `_document.js`, the Docs (https://nextjs.org/docs/advanced-features/custom-document) only give an example of a Class Component. How would I write that as a functional component? I wasn't able to find any examples.

**Comment by akellbl4** [comment]
As I know you have no chance to use the `Document` component as functional. It should be a class component because Next.js wait for it as a class.
